packages/backend/src/services/searchService/integrationTests/utilities/createRequestBuilder.js

```diff
@@ -0,0 +1,59 @@
+module.exports = function createRequestBuilder({
+  aggregationFunction,
+  filterFunction,
+}) {
+  const buildBody = ({
```

Comment:
Might to this later. Its quite likely that it will change a log and I will and integration tests that uses this function which I think will be good enough